source/_integrations/lupusec.markdown

```diff
@@ -23,11 +23,12 @@ The `lupusec` integration allows the user to integrate their Lupusec alarm contr
 Supported units:
 
 - Lupusec XT1
+- Lupusec XT2 Plus
 
 The following devices are supported by the underlying `lupupy` Python library and integrated into Home Assistant.
 
 - **Alarm Control Panel**: Displays the alarm status and controls arming, disarming and home modus.
-- **Binary Sensor**: Displays the status of binary sensors. Currently only Door and window sensors are supported.
+- **Binary Sensor**: Displays the status of binary sensors. Door, window, water and smoke sensors are supported.
```

Comment:
```suggestion
- **Binary Sensor**: Displays the status of binary sensors. Door, window, water, and smoke sensors are supported.
```
Oxford comma as per the [standards](https://developers.home-assistant.io/docs/documenting/standards/).